packages/reports/addon/templates/components/navi-warning.hbs

```diff
@@ -0,0 +1,34 @@
+{{!-- Copyright 2018, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
+{{#if warningEnabled}}
+  {{#liquid-bind warningEnabled class='navi-warning__footer'}}
+    <div class='navi-warning__contents' {{action 'toggleDetails'}}>
+      {{navi-icon 'exclamation-triangle' class='navi-warning__warning-icon'}}
+      <span class='navi-warning__message-text'>Results have missing data.</span>
+      <span class='navi-warning__details-trigger'>
+        {{#if showDetails}}
+          <span class='navi-warning__details-desc'>Click to close.</span>
+          {{navi-icon 'chevron-down' class='navi-warning__arrow-icon'}}
+        {{else}}
+          <span class='navi-warning__details-desc'>Click for more details.</span>
+          {{navi-icon 'chevron-up' class='navi-warning__arrow-icon'}}
+        {{/if}}
+      </span>
+    </div>
+  {{/liquid-bind}}
+  {{#liquid-spacer growDuration=300}}
```

Comment:
Would `liquid-if` work here instead of the `if` inside the `liquid-spacer`?
https://ember-animation.github.io/liquid-fire/helpers/liquid-if

Reply:
I've avoided using `liquid-if` because when the if condition is false, it still creates a non-empty div on the page.